achievements/meeseek.achievement.js

```diff
@@ -0,0 +1,31 @@
+var _ = require('lodash');
+
+var breakingBad = {
+	name: 'I\'m Mr. Meeseeks! Look at me!',
+	check: function(pullRequest, shall) {
+		if (checkIfResolvesManyIssues(pullRequest)) {
+
+			var achievement = {
+				avatar : 'https://i1.sndcdn.com/artworks-000073839997-26zhb5-t500x500.jpg',
+				name: 'I\'m Mr. Meeseeks! Look at me!',
+				short: 'Knock yourselves out just keep your requests simple',
```

Comment:
Missing panctuation: `Knock yourselves out just keep your requests simple` --> `Knock yourselves out. Just keep your requests simple`

I actually love this one (which is the same quote but with the little pause):
```
Knock yourselves out. Just eh-keep your requests simple.
```

choose whatever you want